MesaModel/model/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Facilitate relative imports:
+import os, sys
+
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Can you clarify why this is needed? Could you use e.g. `from .Pupil import Pupil` instead of modifying the path?

Reply:
It facilitates relative imports which allow me to get around problem of importing utils. The first step, `python model/Pupil.py`, works just with the line you suggest, but `python server.py` needs the relative import. Is there a cleaner way of dealing with `utils.py` that I am not noticing?